site/components/context/Vusd.js

```diff
@@ -25,7 +25,7 @@ export const VusdContextProvider = function ({
   const mergeTokenData = (tokens, walletBalances, vusdBalance) =>
     tokens.map(token => ({
       ...token,
-      walletRedeemable: Big(token.redeemableVusd).lt(vusdBalance)
+      walletRedeemable: new Big(token.redeemableVusd).lt(vusdBalance)
```

Comment:
I guess adding the `new` comes from a warning in an eslint rule, but `new` is optional according to big.js docs

Reply:
You are right. Even when it is optional I like to add it for consistency.